app/templates/components/forms/admin/settings/system-form.hbs

```diff
@@ -67,7 +65,7 @@
     <label>
       {{t 'Email'}}
     </label>
-    {{input type='email' name='super_admin_email'}}
+    {{input type='email' name='super_admin_email' value='super@admin.com'}}
```

Comment:
@niranjan94 Please guide regarding from where to fetch the super admin email ID, since it is not the part of the setting model.